test/integration/Adapter/Driver/Pdo/Postgresql/AdapterTest.php

```diff
@@ -4,6 +4,7 @@
 
 use LaminasIntegrationTest\Db\Adapter\Driver\Pdo\AbstractAdapterTest;
 
+#[\AllowDynamicProperties]
```

Comment:
I would prefer you to declare dynamically used properties rather than use this attriburte.
This annotation is hiding code quality issue and it is not suitable for this use case.

Same for all other uses of this attribute below.